### ISSUE TITLE
Document accessibility issues with grouped bar chart

### ISSUE THREAD

**Issue by benjiallen** [benjiallen/accessible-python-dev-survey-2020#5]
Review the grouped bar chart (see Figure 1) used within the [python developers survey](https://www.jetbrains.com/lp/python-developers-survey-2020/). See the section entitled "Python usage with other languages".

Figure 1
<img width="483" alt="2021-04-09_20-47-52" src="https://user-images.githubusercontent.com/16546334/114282238-b0206780-9a10-11eb-94ea-5f6e6f77c895.png" alt="grouped bar chart used in survey results">

The review should be completed with respect to the WCAG 2.1 level A and AA guidelines.

In addition to the accessibility review, consider these questions:

* Could this visualization be easily represented in a data table?
* Does using a data table make the experience more or less efficient? Would important visual data, like trends, be lost in a data table presentation?
* Would an alternative visualization do a better job of conveying the data? If so, why?
* Does the chart title and supporting text do a good enough job in explaining the data within the visualization?
